ci/custom_stemcell/tasks/capture-stemcell.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
```

Comment:
Please try to pass shell check and shell format for each bash script, @EmilyEmily knows it :)